ci-operator/step-registry/edge-nodes/machineset-manifests/edge-nodes-machineset-manifests-commands.sh

```diff
@@ -7,9 +7,24 @@ set -o pipefail
 REGION="${LEASED_RESOURCE}"
 export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
 
-edge_zone_subnet_id=$(head -n 1 "${SHARED_DIR}/edge_zone_subnet_id")
-edge_zone_name=$(head -n 1 "${SHARED_DIR}/edge-zone-name.txt")
-edge_zone_group_name=$(head -n 1 "${SHARED_DIR}"/edge-zone-group-name.txt)
+if [[ ${EDGE_ZONE_TYPE} == "local-zone" ]] || [[ ${EDGE_ZONE_TYPE} == "wavelength-zone" ]]; then
```

Comment:
I am thinking if it is possible to make the above two step to be simple as much as possible for a dedicated purpose, to reduce complexity because of compatibility for different scenarioes (localzone, wavelength, and outpost) .

Reply:
> above two step

which two steps?

Reply:
Oh, I place the comments on wrong lines, I meant:
```
  - ref: edge-nodes-machineset-manifests
  - ref: edge-nodes-provision-localzone-day2
```

Reply:
Yes, I have the similar concern.

Currently:
* the step `edge-nodes-machineset-manifests` is being used by pre-install (e.g. `cucushift-installer-rehearse-aws-ipi-disconnected-private-network-mtu-localzone-provision`) and post install (e.g. day2 cases)
* the chain `edge-nodes-provision-localzone-day2` is being used by WLZ and LZ, _the LZ/WLZ subnets creation is included_
* the outpost uses single steps `edge-nodes-machineset-manifests` and `edge-nodes-provision-localzone-day2` in provision chain. 

How about remove `- chain: aws-provision-localzone-subnet` from chain `edge-nodes-provision-localzone-day2`, this will result:
1. In each WLZ/LZ day 2 workflow
```
- chain: aws-provision-localzone-subnet
...
- ref: ipi-install-install
... 
- chain: edge-nodes-provision-localzone-day2
```
2. In Outpost day 2 workflow
```
- ref: ipi-install-install
...
- chain: edge-nodes-provision-localzone-day2
```
3. For pre-install
```
- ref: edge-nodes-machineset-manifests
- ref: ipi-install-install
```